projects/ngx-charts-on-fhir/src/lib/data-layer-list/annotation-list/annotation-list.component.spec.ts

```diff
@@ -21,4 +21,87 @@ describe('DatasetAnnotationListComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('when onAnnotationsChange called it should emit event', () => {
```

Comment:
FYI: The jasmine function is named `it` so you can use it as the first word in the sentence. Sort of like a writing prompt to help us write BDD-style expectations. For example `it('should do X', ...)` reads as "it should do X".

When testing a class, I like to create a `describe()` for each method. For example:
```
describe('AnnotationListComponent', () => {
  describe('onAnnotationsChange', () => {
    it('should emit annotationsChange event', ...);
  });
});
```